actor-testkit-typed/src/test/java/jdocs/org/apache/pekko/actor/testkit/typed/javadsl/SyncTestingExampleTest.java

```diff
@@ -14,22 +14,23 @@
 package jdocs.org.apache.pekko.actor.testkit.typed.javadsl;
 
 // #imports
+import static org.junit.Assert.assertEquals;
+
+import com.typesafe.config.Config;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Optional;
 import org.apache.pekko.actor.testkit.typed.CapturedLogEvent;
 import org.apache.pekko.actor.testkit.typed.Effect;
 import org.apache.pekko.actor.testkit.typed.javadsl.BehaviorTestKit;
 import org.apache.pekko.actor.testkit.typed.javadsl.TestInbox;
 import org.apache.pekko.actor.typed.*;
 import org.apache.pekko.actor.typed.javadsl.*;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Optional;
-
-import com.typesafe.config.Config;
-import org.slf4j.event.Level;
-// #imports
```

Comment:
Is the code ordered the old way to highlight just the main imports? The Paradox plugin can be retrieve code based on comments in this format.